testsuite/features/core_ubuntu_salt.feature

```diff
@@ -1,48 +1,55 @@
 # Copyright (c) 2017-2018 SUSE LLC
 # Licensed under the terms of the MIT license.
 #
-# 1) register an Ubuntu minion via GUI
+# 1) register an Ubuntu minion via GUI (without proxy)
 # 2) run a remote command
 # 3) try an openscap scan
 
 Feature: Be able to bootstrap an Ubuntu minion and do some basic operations on it
 
+#@ubuntu_minion
+#  Scenario: Install Salt service from Ubuntu minion
+#    When I install Salt packages from "ubuntu-minion"
+#    And I start salt-minion on "ubuntu-minion"
+
 @ubuntu_minion
   Scenario: Bootstrap an Ubuntu minion
     Given I am authorized
-    When I go to the bootstrapping page
-    Then I should see a "Bootstrap Minions" text
-    When I enter the hostname of "ubuntu-minion" as "hostname"
+    And I go to the bootstrapping page
+    And I should see a "Bootstrap Minions" text
+    And I enter the hostname of "ubuntu-minion" as "hostname"
     And I enter "22" as "port"
     And I enter "root" as "user"
     And I enter "linux" as "password"
     And I select "1-UBUNTU-TEST" from "activationKeys"
-    And I select the hostname of the proxy from "proxies"
-    And I click on "Bootstrap"
-    And I wait until I see "Successfully bootstrapped host! " text
+    When I click on "Bootstrap"
+    Then I wait until I see "Successfully bootstrapped host!" text
     And I navigate to "rhn/systems/Overview.do" page
     And I wait until I see the name of "ubuntu-minion", refreshing the page
     And I wait until onboarding is completed for "ubuntu-minion"
+    And I query latest Salt changes on ubuntu system "ubuntu-minion"
 
-@proxy
 @ubuntu_minion
-  Scenario: Check connection from Ubuntu minion to proxy
+  Scenario: Schedule an OpenSCAP audit job for the Ubuntu minion
```

Comment:
This should not be in a core feature.

I suggest you commit "as is", and let @lkotek move it to a secondary feature for you. He just has to do as he did for CentOS.

Alternatively, take your inspiration from @lkotek's work, and split this yourself.

Reply:
Totally agree, we will address that in a separate PR.